Tasks/Korotkevich/Task6/task_4.py

```diff
@@ -0,0 +1,23 @@
+import os
+p = os.path.join('Md-PT-64-23', 'Tasks', 'Korotkevich', 'Task6')
+os.chdir(p)
+
+import time
+
+def create_file(func):
+    def wrapper(*args):
+        global time
+        start = time.time()
+        func(*args)
+        finish = time.time()
+        result = func(*args)
+        time = finish - start
```

Comment:
by doing so you overriding the time module, it's very bad approach

moreover, you are using global here without any reason whatsoever, you write the result iin a file, you do not need to store it globally at all